src/btree.c

```diff
@@ -3246,42 +3253,52 @@ btree_print_node(platform_log_handle *log_handle,
       platform_log(log_handle, "-------------------\n");
       return;
    }
-   btree_node_get(cc, cfg, node, PAGE_TYPE_BRANCH);
-   btree_print_locked_node(log_handle, cfg, node->addr, node->hdr);
+   btree_node_get(cc, cfg, node, type);
```

Comment:
Use the passed-in `type` argument in preference to hard-coding as `PAGE_TYPE_BRANCH` as done on old L3249.